src/infra/tokenizer/token-manager.ts

```diff
@@ -0,0 +1,52 @@
+import * as E from "fp-ts/lib/Either";
+import * as TE from "fp-ts/lib/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+
+import { FiscalCode } from "@pagopa/io-functions-services-sdk/FiscalCode";
+import { config } from "../../app/config";
+import { tokenizerApiClient } from "./client";
+import { SignerId } from "../../signer/signer";
```

Comment:
```suggestion
import { config } from "../../app/config";
import { SignerId } from "../../signer/signer";
import { tokenizerApiClient } from "./client";
```
`../../signer/signer` import should occur before import of `./client`